docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  app:
+    container_name: app
+    build:
+      context: .
+    command: python provenance_store/manage.py runserver 0.0.0.0:8000
+    #command: tail -f /dev/null
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    volumes:
+      - .:/app
+    networks:
+      - amplify_provenance
+    restart: unless-stopped
+    depends_on:
+      - db
+
+  db:
+    container_name: db
+    image: postgres:17.2-alpine
+    env_file:
+      - .env
+    volumes:
+      - /srv/amplify-provenance:/var/lib/postgresql/data
```

Comment:
This is hardcoded--suggest that it either come from `.env` or be replaced by a Docker volume